apps/jubmoji-quest/src/pages/collect/index.tsx

```diff
@@ -18,8 +18,8 @@ import { Onboarding } from "@/components/Onboarding";
 import {
   JubmojiCardProps,
   getJubmojiCardByPubIndex,
-  useCards,
-} from "@/hooks/useCards";
+  useFetchCards,
+} from "@/hooks/useFetchCards";
 import Image from "next/image";
 
 const OnboardSection = () => {
```

Comment:
![IMG_6837](https://github.com/jubmoji/jubmoji.quest/assets/6290822/6482a9ec-f911-4a4c-a406-bf260b9f7ade)

Text is now off center for onboarding for some reason